src/pages/rt-fees-schedule/forms/FeesDetailsForm.js

```diff
@@ -0,0 +1,156 @@
+import { Grid, Box, Checkbox } from '@mui/material'
+import { useFormik } from 'formik'
+import { useContext, useEffect, useState } from 'react'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import FormShell from 'src/components/Shared/FormShell'
+import * as yup from 'yup'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { DataSets } from 'src/resources/DataSets'
+import { RemittanceOutwardsRepository } from 'src/repositories/RemittanceOutwardsRepository'
+
+const FeesDetailsForm = ({ store, labels, editMode, maxAccess }) => {
+  const { recordId: pId } = store
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  useEffect(() => {
+    getGridData(pId)
+  }, [pId])
+
+  function getGridData(scheduleId) {
+    formik.setValues({
```

Comment:
what the purpose of this? if u already set formik initial values

Reply:
new Row